charts/mod-authtoken/values.yaml

```diff
@@ -109,6 +109,7 @@ extraEnv: |
 dbConnectParameters: db-connect-modules 
 # kafkaConnectParameters: db-connect-modules
 # awsConnectParameters: s3-credentials
+rtrEnabled: false
```

Comment:
Will it be overwritten in config? 
https://github.com/folio-org/pipelines-shared-library/pull/423/files
it seems to be duplication of the value, we'll have both  rtrEnabled: false  && rtrEnabled: true, right after the 1st one.